build_system/seqan3-config.cmake

```diff
@@ -599,6 +599,15 @@ if (NOT ${FIND_NAME}_FIND_QUIETLY)
     message (STATUS "${ColourBold}Found SeqAn3:${ColourReset} ${SEQAN3_BASEDIR}/seqan3 (found version \"${SEQAN3_VERSION_STRING}\")")
 endif ()
 
+separate_arguments (SEQAN3_CXX_FLAGS_LIST UNIX_COMMAND "${SEQAN3_CXX_FLAGS}")
+string(REGEX REPLACE ";( )*[-/]D" ";" SEQAN3_DEFINITIONS_LIST ";${SEQAN3_DEFINITIONS};")
+
+add_library (seqan::seqan3 INTERFACE IMPORTED)
```

Comment:
why did you call it "seqan::seqan3" and not just "seqan2?

Reply:
it's cool to have a namespace we could later have 
```cmake
add_library (seqan3::seqan3 INTERFACE IMPORTED)
add_library (seqan3::zlib INTERFACE IMPORTED)
add_library (seqan3::bzip INTERFACE IMPORTED)
```

And the user could pull the changes in.

Reply:
I would suggest the name change to `seqan3::seqan3`

Reply:
What's the point of of seqan3::zlib? Wouldnt that be part of seqan3 anyway? :thinking: